k2/csrc/properties.h

```diff
@@ -32,8 +32,7 @@ enum Properties {
   `fsa` is valid if:
   1. it is empty, if not, it contains at least two states.
   2. only kFinalSymbol arcs enter the final state.
-  3. every state contains at least one arc except the final state.
```

Comment:
seems this is not a reasonable requirement (case we dont do `Connect` after some algorthims).

Reply:
You're right, that can be removed, it was an early draft.

Reply:
i will remove